src/processor/operator/aggregate/aggregate_hash_table.cpp

```diff
@@ -804,9 +792,35 @@ uint64_t PartitioningAggregateHashTable::append(const std::vector<ValueVector*>&
     return numFlatTuples;
 }
 
-void PartitioningAggregateHashTable::mergeAll() {
-    partitioningData->appendTuples(*factorizedTable,
-        tableSchema.getColOffset(tableSchema.getNumColumns() - 1));
+bool hasSpace(uint64_t startingNumTuples, uint64_t newNumTuples, uint64_t currentMax) {
```

Comment:
Should this be `hasSpace` or `outOfSpace`?

Reply:
Yes, I guess I got that inverted somehow